packages/console/src/components/Tooltip/index.tsx

```diff
@@ -9,14 +9,15 @@ type Props<T> = {
   content: ReactNode;
   domRef: RefObject<Nullable<T>>;
   className?: string;
+  isAuto?: boolean;
```

Comment:
what about change the name into "isVisible", if set to true, means always visible and remove mouse event handler.